.cirrus.yml

```diff
@@ -708,6 +703,32 @@ podman_machine_task:
     always: *int_logs_artifacts
 
 
+podman_machine_aarch64_task:
+    name: *std_name_fmt
+    alias: podman_machine_aarch64
+    only_if: *not_tag_branch_build_docs
+    depends_on:
+        - build_aarch64
+        - validate_aarch64
+        - local_integration_test
+        - remote_integration_test
+        - container_integration_test
+        - rootless_integration_test
+    ec2_instance:
+        <<: *standard_build_ec2_aarch64
+        type: "${EC2_INST_TYPE}"  # Bare-metal instance is required
```

Comment:
I could be mistaken, but in my copy of the file, `&standard_build_ec2_aarch64` already defines this line to use the variable.  Not a big deal to have it here, but it's unnecessary duplication.

Reply:
ah ok, my bad. Removing that ..

Reply:
done